verilog/analysis/verilog_linter_configuration_test.cc

```diff
@@ -803,6 +825,20 @@ TEST(ConfigureFromOptionsTest, Basic) {
   EXPECT_TRUE(status.ok());
 }
 
+TEST(ConfigureFromOptionsTest, LoadFromNonExistingFile) {
+  LinterConfiguration config;
+
+  LinterOptions options = {.ruleset = RuleSet::kAll,
+                           .rules = RuleBundle(),
+                           .config_file = "non-existent-file.txt",
+                           .rules_config_search = false,
+                           .linting_start_file = "filename",
+                           .waiver_files = "filename"};
```

Comment:
maybe a different fake file name for .waiver_files

Reply:
this is mostly for symmetry with the other tests (that filename is not looked at)